test/functional/terminal/tui_spec.lua

```diff
@@ -875,6 +875,11 @@ describe('TUI', function()
       {3:-- TERMINAL --}                                    |
     ]=])
   end)
+
+  it("does not crash on successive <ESC> inputs #13551", function()
+    command([[call chansend(b:terminal_job_id, repeat("\<ESC>", 257))]])
+    screen:expect_unchanged()
```

Comment:
I think that the recommendation is using `alive()` here?

Reply:
I think what we need to check is whether nvim in the terminal is alive and accepting input or not, so I did the following.

```lua
command([[call chansend(b:terminal_job_id, ":echo 'alive'\<CR>")]])
screen:expect{any='alive'}
```